common/men_at_arms_types/temp_draft_doc.txt

```diff
@@ -0,0 +1,575 @@
+		hills = { damage = 25 toughness = 25 pursuit = 10 }
+		drylands = { damage = 25 toughness = 25 pursuit = 10 }
+	}
+
+	counters = {
+		pikemen = 1
+		massive_infantry = 1
+		heavy_infantry = 1
+	}
+
+	buy_cost = { gold = cost_120_recruitment_cost }
+	low_maintenance_cost = { gold = cost_120_low_maint_cost }
+	high_maintenance_cost = { gold = cost_120_high_maint_cost }
+	
+	stack = 75
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = thornweaver
+}
+#Titanforged
+iron_giant = {
+	type = massive_infantry
+	
+	damage = 125
+	toughness = 136
+	pursuit = 5
+	screen = 50
+	
+	terrain_bonus = {
+		mountains = { damage = 15 toughness = 10 screen = 10}
+	}
+
+	counters = {
+		skirmishers = 2
+		archers = 2
+	}
+	
+	buy_cost = { gold = cost_180_recruitment_cost }
+	low_maintenance_cost = { gold = cost_180_low_maint_cost }
+	high_maintenance_cost = { gold = cost_180_high_maint_cost }
+	
+	stack = 50
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = iron_giant
+}
+#Tuskarr
+trapper = {
+	type = archers
+	
+	damage = 30
+	toughness = 25
+	pursuit = 25
+	screen = 20
+	
+	terrain_bonus = {
+		mountains = { damage = 10 toughness = 10 pursuit = 10 screen = 10 }
+		taiga = { damage = 10 toughness = 10 pursuit = 10 screen = 10 }
+		desert = { damage = -10 toughness = -10 pursuit = -10 screen = -10 }
+		desert_mountains = { damage = -10 toughness = -10 pursuit = -10 screen = -10 }
+		oasis = { damage = -10 }
+		jungle = { damage = 10 thoughness = -10 }
+	}
+
+	winter_bonus = {
+		mild_winter = { damage = 10 toughness = 10 }
+		normal_winter = { damage = 8 toughness = 8 }
+		harsh_winter = { damage = 6 toughness = 6 }
+	}
+
+	counters = {
+		skirmishers = 1
+		light_aerial = 2
+		light_cavalry = 1
+	}
+
+	buy_cost = { gold = cost_105_recruitment_cost }
+	low_maintenance_cost = { gold = cost_105_low_maint_cost }
+	high_maintenance_cost = { gold = cost_105_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = trapper
+}
+
+turtle_rider = {
+	type = heavy_cavalry
+	
+	damage = 175
+	toughness = 180
+	pursuit = 15
+	screen = 15
+	
+	terrain_bonus = {
+		mountains = { damage = 20 toughness = 20 pursuit = 20 screen = 20 }
+		taiga = { damage = 20 toughness = 20 pursuit = 20 screen = 20 }
+		desert = { damage = -5 toughness = -5 screen = -5 }
+		desert_mountains = { damage = -10 toughness = -10 screen = -10 }
+		oasis = { damage = -5 }
+		jungle = { damage = -8 thoughness = -8 }
+	}
+
+	winter_bonus = {
+		mild_winter = { damage = 5 toughness = 5 }
+		normal_winter = { damage = 4 toughness = 4 }
+		harsh_winter = { damage = 4 toughness = 4 }
+	}
+
+	counters = {
+		casters = 1
+		archers = 1
+	}
+
+	buy_cost = { gold = cost_240_recruitment_cost }
+	low_maintenance_cost = { gold = cost_240_low_maint_cost }
+	high_maintenance_cost = { gold = cost_240_high_maint_cost }
+	
+	stack = 50
+	ai_quality = { value = @cultural_maa_extra_ai_score }
+	icon = turtle_rider
+}
```

Comment:
Turtle riders are not fixed (small bonus/malus is disproportionate to base value). Plus, balancing according to the spreadsheet is neither balanced regarding cost/value nor cost/base unit cost.

Reply:
I agree on small bonuses and maluses. They're barely noticeable. Malus/bonus should be at least 10% from the base stat.